src/components/CohortBuilder/Results.js

```diff
@@ -7,7 +7,7 @@ import Row from 'uikit/Row';
 import ViewLink from 'uikit/ViewLink';
 import styled from 'react-emotion';
 import { H2 } from 'uikit/Headings';
-import Table from './Table';
+import ParticipantsTableView from './Table';
```

Comment:
would it be too much if we renamed `/Table` to `/ParticipantsTableView`? :)

Reply:
The filename? Np.